arcsi/api/show.py

```diff
@@ -61,6 +62,12 @@ class ShowDetailsSchema(Schema):
         ),
         required=True,
     )
+    tags = fields.List(
```

Comment:
It would be better to return the id of the tag here too and we could expand the schemas with the tag field as well (as I see it is enough to modify show_archive_schema since it is the one that supposed to be used for the show subpages).